core/templates/dev/head/components/AnswerGroupEditorDirective.js

```diff
@@ -30,6 +30,7 @@ oppia.directive('answerGroupEditor', [
           '&onSaveAnswerGroupCorrectnessLabel',
         outcome: '=',
         suppressWarnings: '&',
+        isCorrectnessFeedbackEnabled: '=',
```

Comment:
Looks like this is a variable, not a function, so don't start with a verb. Call it correctnessFeedbackIsEnabled instead.

EDIT: on second thoughts, don't pass in a new attr at all. This directive should just depend on the explorationCorrectnessFeedbackService directly.

Reply:
Yep ! That would be better. I will commit the changes with directive directly using the explorationCorrectnessFeedbackService.

Reply:
Done!